### PR TITLE
Integrating uptime-tracker ( https://github.com/SUSE/uptime-tracker/ )

### PR DESCRIPTION
**Pull request by chajain**
Integrated uptime-tracker ( https://github.com/SUSE/uptime-tracker ) in cmd folder
Added the suse-uptime-tracker services in spec file and build/packaging folder

https://jira.suse.com/browse/CSE-503

### CODE REVIEW

File: build/packaging/suseconnect-ng.spec [hunk 1]
```diff
@@ -139,13 +141,16 @@ install -D -m 644 SUSEConnect.example %{buildroot}%{_sysconfdir}/SUSEConnect.exa
 # Install the SUSEConnect --keepalive timer and service.
 install -D -m 644 build/packaging/suseconnect-keepalive.timer %{buildroot}/%{_unitdir}/suseconnect-keepalive.timer
 install -D -m 644 build/packaging/suseconnect-keepalive.service %{buildroot}/%{_unitdir}/suseconnect-keepalive.service
+install -D -m 644 build/packaging/suse-uptime-tracker.timer %{buildroot}/%{_unitdir}/suse-uptime-tracker.timer
+install -D -m 644 build/packaging/suse-uptime-tracker.service %{buildroot}/%{_unitdir}/suse-uptime-tracker.service
 ln -sf service %{buildroot}/%{_sbindir}/rcsuseconnect-keepalive
+ln -sf service %{buildroot}/%{_sbindir}/rcsuse-uptime-tracker
```
Comment (guangyee) [inline build/packaging/suseconnect-ng.spec:147]: I don't think we have `rcsuse-uptime-tracker`.

File: cmd/suse-uptime-tracker/uptime_tracker.go [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+
+	flag.BoolVar(&version, "version", false, "")
+
+	flag.Parse()
+	if version {
+		fmt.Println(getShortenedVersion())
+		os.Exit(0)
+	}
+
+	uptimeLogs, err := readUptimeLogFile(uptimeCheckLogsFilePath)
```
Comment (ngetahun) [inline cmd/suse-uptime-tracker/uptime_tracker.go:143]: Please refactor this into it's own function. It is much easier to test and alter later.

Comment (chajain) [reply cmd/suse-uptime-tracker/uptime_tracker.go:143]: readUptimeLogFile is already a function. Can you please clarify, what I need to refactor?

Comment (ngetahun) [reply cmd/suse-uptime-tracker/uptime_tracker.go:143]: I meant the block of code

**Comment by jeremy-moffitt** [comment]
@ngetahun please let @chajain know if her changes here have addressed your feedback or if more is required. Thanks!

**Comment by likhitha77** [comment]
[Display systems uptime data to customers on SCC](https://trello.com/c/3afCxV3i/3457-display-systems-uptime-data-to-customers-on-scc)

**Review by ngetahun — APPROVED** [review]
LGTM!

**Pull request merged**